docs/data/base/components/switch/switch.md

```diff
@@ -1,18 +1,17 @@
 ---
 product: base
-title: React Switch component
+title: Unstyled React Switch component and hook
 components: SwitchUnstyled
 githubLabel: 'component: switch'
 waiAria: https://www.w3.org/TR/wai-aria-practices/#switch
 packageName: '@mui/base'
 ---
 
-# Switch
+# Unstyled switch
 
-<p class="description">The Switch component lets you toggle the state of a single setting on or off.</p>
+<p class="description">The `SwitchUnstyled` component provides users with a switch for toggling between two mutually exclusive states.</p>
 
-Switches can be used on either desktop or mobile but it has been the preferred way for toggling settings in the latter.
-To make the switch accessible, you should ensure the corresponding label reflects the current state of the switch.
+Switches can be used on either desktop or mobile, but they are the preferred method for toggling on mobile UIs.
```

Comment:
> they are the preferred method

Preferred by who? Us? Material Design? 😅

Reply:
Good question! Lemme see if I can find a source to cite for this claim!

Reply:
Hmm I think that this historically comes from Material Design - https://material.io/components/switches#usage but I don't think we should generalize this on the base package. For example, if we take as an example the aria specification on switch - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/switch_role it doesn't say anything like that.

Reply:
Exactly! I think we can just ditch that sentence altogether, parting from the assumption that "can be used on either desktop or mobile" is a given for every component.

Reply:
Roger that!